components/layout/navbar/profile-menu/profile-modal.tsx

```diff
@@ -0,0 +1,57 @@
+"use client";
+
+import { MenuContent } from "@/types/ui-content";
+import { useEffect, useRef, RefObject } from "react";
+import { useSession } from "next-auth/react";
+import { ProfileItem } from "./profile-item";
+import Image from "next/image";
+
+// TODO: Make it work for contact us
```

Comment:
is this still TODO?

Reply:
i think it would be better to created another component for contact us